udata_front/theme/gouvfr/datagouv-components/src/locales/it.json

```diff
@@ -83,5 +83,6 @@
   "Archived": "Archived",
   "{n} reuses": "{n} reuses",
   "{n} favorites": "{n} favorites",
+  "Published {date}": "Published {date}",
```

Comment:
Did you add these ? Or are they coming from master ?

Reply:
It came from master

Reply:
After check, it comes from me